ci-operator/step-registry/cucushift/installer/check/machine-name/cucushift-installer-check-machine-name-commands.sh

```diff
@@ -0,0 +1,35 @@
+!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+
+# Machines
+err_output=$(mktemp)
+machine_output=$(mktemp)
+oc get machine -n openshift-machine-api -owide --no-headers 1>${machine_output} 2>${err_output}
```

Comment:
Does this script only use to check master node name? If so, how about to only filter master node via below command?
```
oc get machine -n openshift-machine-api --selector machine.openshift.io/cluster-api-machine-type=master --no-headers
```

Reply:
OK, I will udpate, thanks